packages/core/src/test/amazonqGumby/resources/files/humanInTheLoop/downloadResults/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
test "resources" should live in testFixtures https://github.com/aws/aws-toolkit-vscode/tree/f9d94712dfbb6a6be5597be72f2f5201a7913c52/packages/core/src/testFixtures

Reply:
Is there a functional reason for this?

We are trying to keep our fixtures and resources close to our code since upstream refactors have happened multiple times the last couple months. For risk mitigation, we want to keep our resources/code away from shared spaces when possible.

Reply:
What's the risk here? If a refactor breaks your tests... we will know because the tests fail.  Not sure if there is a technical reason though.

Reply:
The risk is having changes to our codebase without no visibility. This has happened several times in the past. 

Yet, the inverse is not true. We cannot make changes to the codebase without your team approval.

Reply:
The risk is having changes to our codebase without no visibility. This has happened several times in the past. 

Yet, the inverse is not true. We cannot make changes to the codebase without your team approval.